internal/service/encryptionatrest/data_source_schema.go

```diff
@@ -0,0 +1,184 @@
+						Sensitive:           true,
+						Description:         "Unique 36-hexadecimal character string that identifies an Azure application associated with your Azure Active Directory tenant.",
+						MarkdownDescription: "Unique 36-hexadecimal character string that identifies an Azure application associated with your Azure Active Directory tenant.",
+					},
+					"enabled": schema.BoolAttribute{
+						Computed:            true,
+						Description:         "Flag that indicates whether someone enabled encryption at rest for the specified  project. To disable encryption at rest using customer key management and remove the configuration details, pass only this parameter with a value of `false`.",
+						MarkdownDescription: "Flag that indicates whether someone enabled encryption at rest for the specified  project. To disable encryption at rest using customer key management and remove the configuration details, pass only this parameter with a value of `false`.",
+					},
+					"key_identifier": schema.StringAttribute{
+						Computed:            true,
+						Sensitive:           true,
+						Description:         "Web address with a unique key that identifies for your Azure Key Vault.",
+						MarkdownDescription: "Web address with a unique key that identifies for your Azure Key Vault.",
+					},
+					"key_vault_name": schema.StringAttribute{
+						Computed:            true,
+						Description:         "Unique string that identifies the Azure Key Vault that contains your key.",
+						MarkdownDescription: "Unique string that identifies the Azure Key Vault that contains your key.",
+					},
+					"require_private_networking": schema.BoolAttribute{
+						Computed:            true,
+						Description:         "Enable connection to your Azure Key Vault over private networking.",
+						MarkdownDescription: "Enable connection to your Azure Key Vault over private networking.",
+					},
+					"resource_group_name": schema.StringAttribute{
+						Computed:            true,
+						Description:         "Name of the Azure resource group that contains your Azure Key Vault.",
+						MarkdownDescription: "Name of the Azure resource group that contains your Azure Key Vault.",
+					},
+					"secret": schema.StringAttribute{
+						Computed:            true,
+						Sensitive:           true,
+						Description:         "Private data that you need secured and that belongs to the specified Azure Key Vault (AKV) tenant (**azureKeyVault.tenantID**). This data can include any type of sensitive data such as passwords, database connection strings, API keys, and the like. AKV stores this information as encrypted binary data.",
+						MarkdownDescription: "Private data that you need secured and that belongs to the specified Azure Key Vault (AKV) tenant (**azureKeyVault.tenantID**). This data can include any type of sensitive data such as passwords, database connection strings, API keys, and the like. AKV stores this information as encrypted binary data.",
```

Comment:
```suggestion
						Description:         "Private data to secure that belongs to the specified Azure Key Vault (AKV) tenant (**azureKeyVault.tenantID**). AKV stores this information as encrypted binary data.",
						MarkdownDescription: "Private data to secure that belongs to the specified Azure Key Vault (AKV) tenant (**azureKeyVault.tenantID**). AKV stores this information as encrypted binary data.",
```
I don't think we need to prescribe use-cases for this field. The description of what it does is sufficiently suggestive to users.

Reply:
Same as above, as shared in the slack thread since this is being fetched directly from the API spec, would like to keep these in alignment, have applied the necessary comments though, please lmk if you still have any concerns:)